dspace/config/spring/api/suggestions.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:context="http://www.springframework.org/schema/context"
+    xmlns:util="http://www.springframework.org/schema/util"
+    xsi:schemaLocation="http://www.springframework.org/schema/beans
+           http://www.springframework.org/schema/beans/spring-beans-2.5.xsd
+           http://www.springframework.org/schema/context
+           http://www.springframework.org/schema/context/spring-context-2.5.xsd
+           http://www.springframework.org/schema/util 
+           http://www.springframework.org/schema/util/spring-util-3.0.xsd">
+
+    <context:annotation-config /> <!-- allows us to use spring annotations in beans -->
+
+    <!-- This is defined in the solr-services.xml
+	<bean id="org.dspace.app.nbevent.service.NBEventService" 
+	   class="org.dspace.app.nbevent.service.impl.NBEventServiceImpl" /> -->
+ 
+	 <util:map id="suggestionProviders" map-class="java.util.HashMap" 
+	          key-type="java.lang.String" value-type="org.dspace.app.suggestion.SuggestionProvider">
+	    <entry key="openaire" value-ref="OpenairePublicationLoader" />
+	</util:map>
+
+    <bean id="OpenairePublicationLoader" class="org.dspace.app.suggestion.openaire.PublicationLoader">
+        <property name="sourceName" value="openaire" />
+        <property name="primaryProvider" ref="openaireLiveImportDataProviderByAuthor" />
+        <property name="otherProviders">
+            <list>
+                <ref bean="openaireLiveImportDataProviderByTitle"/>
+            </list>
+        </property>
+        <property name="names">
+            <list>
+                <value>dc.title</value>
+            </list>
+        </property>
+        <property name="pipeline">
+            <list>
+                <bean
+                    class="org.dspace.app.suggestion.openaire.AuthorNamesScorer">
+                    <property name="contributorMetadata">
+                        <list>
+                            <value>dc.contributor.author</value>
+                        </list>
+                    </property>
+                    <property name="names">
+                        <list>
+                            <value>dc.title</value>
+                        </list>
+                    </property>
+                </bean>
+                <bean
+                    class="org.dspace.app.suggestion.openaire.DateScorer">
+                    <property name="birthDateMetadata" value="person.birthDate" />
+                    <property name="publicationDateMetadata" value="dc.date.issued" />
+                    <property name="maxDateMetadata" value="person.range.maxdate" />
```

Comment:
@paulo-graca **maxDateMetadata** & **minDateMetadata** are used to configure metadata to retrieve an arbitrary date range.
The two configured metadata fields (in this case person.range.maxdate & person.range.mimdate) can be used to define this custom date range to be used during the Date Score calculations for the suggestions.
These fields are not mandatory and if not set/configured the calculation is performed using the **person.birthDate** metadata or another metadata configurable for the education date.

@paulo-graca this Score mechanism is not something mandatory in order to approve the suggestion but it's just a helpful feature that is actually displaying the "accuracy" for each suggestion (this value is different for each Person).
The previous discussion point (https://github.com/DSpace/dspace-angular/pull/1638#issuecomment-1936061780 @frabacche ) was that the DateScore was always zero. We thought adding this metadata might be really useful for testing out the feature.

I know these metadata are not available in the person schema https://schema.org/Person but we didn't find any existing appropriate metadata. If you @paulo-graca or @tdonohue have a suggestion for the metadata to be used it would be great.

If you prefer we can revert these changes, or we can just leave this configuration only adapting the metadata if it makes sense to you.

Reply:
I agree with @paulo-graca that I find these `maxDate` and `minDate` fields very confusing.  Despite the explanations, I still do not understand their puprose.

Are they set globally for *all researchers*? 

Are they supposed to be uniquely set per researcher?

If the latter, how is user of DSpace (logged in researcher) supposed to know what to set `maxDate` and `minDate` to?  Are these dates supposed to be the person's birth/death dates?  Are they supposed to be the date they started publishing & the date they stopped publishing? Are they supposed to be the years they went to school ?  Or maybe the years they worked? 

Currently there is no information provided in the user interface on how to use these fields.  I understand they somehow impact the behavior of the `DateScorer`, but that isn't something a normal DSpace user will understand.

I see two options:
* Either we need a better description of these fields & what dates a user should add to them (so that we can make this feature easier to use)
* Or, we remove these fields entirely and just use the person's birthDate in the `DateScorer`.

Reply:
Closing this feedback as resolved. @frabacche removed the maxDate and minDate from this PR in [c2f0c6f](https://github.com/DSpace/DSpace/pull/8280/commits/c2f0c6f331a752e24d7be88ead61989e3b93f2b6)